### PR TITLE
Updating ASP.NET Core extension version

### PR DESCRIPTION
**Pull request by fabiocav**
Updating extension version for release.

### Pull request checklist

* [x] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [x] My changes **should not** be added to the release notes for the next release
  * [ ] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

**Review by mattchenderson — APPROVED** [review]
Do we also need to update release notes here? Or was that already in the previous PR?

File: extensions/Worker.Extensions.Http.AspNetCore/src/Worker.Extensions.Http.AspNetCore.csproj [hunk 1]
```diff
@@ -6,7 +6,8 @@
     <Description>ASP.NET Core extensions for .NET isolated functions</Description>
 
     <!--Version information-->
-    <VersionPrefix>1.3.3</VersionPrefix>
+    <VersionPrefix>2.0.0</VersionPrefix>
+    <VersionSuffix>-preview1</VersionSuffix>
```
Comment (kshyju) [inline extensions/Worker.Extensions.Http.AspNetCore/src/Worker.Extensions.Http.AspNetCore.csproj:10]: Unrelated to this change. But should we pin to specific versions other package references below (Worker.Extensions.Abstraction etc.,) ?

**Review by kshyju — APPROVED** [review]
Echoing the same question Matthew raised about TargetFramework value update.

File: extensions/Worker.Extensions.Http.AspNetCore/src/Worker.Extensions.Http.AspNetCore.csproj [hunk 2]
```diff
@@ -6,7 +6,8 @@
...
     <TargetFramework>net6.0</TargetFramework>
```
Comment (mattchenderson) [inline extensions/Worker.Extensions.Http.AspNetCore/src/Worker.Extensions.Http.AspNetCore.csproj:11]: Should the TFM also be updated to reflect use of the newer types?

Comment (fabiocav) [reply extensions/Worker.Extensions.Http.AspNetCore/src/Worker.Extensions.Http.AspNetCore.csproj:11]: I don't want to combine that change with this, but that will likely change for 2.0. Let's discuss whether we want that change in place before the first drop.

File: extensions/Worker.Extensions.Http.AspNetCore/src/Worker.Extensions.Http.AspNetCore.csproj (hunk 1 above)
Comment (fabiocav) [reply extensions/Worker.Extensions.Http.AspNetCore/src/Worker.Extensions.Http.AspNetCore.csproj:10]: This shouldn't be required, but I would like to follow up on that separately. Let's get the proper version in place and follow up.

**Comment by fabiocav** [comment]
About @mattchenderson's comment:

> Do we also need to update release notes here? Or was that already in the previous PR?

Discussed offline., It was not in the original PR with the changes, but will be applied when the release is tagged.

**Pull request merged**